src/Services/Air/AirParser.js

```diff
@@ -531,6 +520,55 @@ const airGetTicket = function (obj) {
   return getTicketFromEtr.call(this, etr, obj);
 };
 
+function airGetTicketsErrorHandler(rsp) {
+  let errorInfo;
+  let code;
+  try {
+    errorInfo = (
+      rsp.detail[`common_${this.uapi_version}:ErrorInfo`]
+      || rsp.detail['air:AvailabilityErrorInfo']
+    );
+    code = errorInfo[`common_${this.uapi_version}:Code`];
+  } catch (err) {
+    throw new RequestRuntimeError.UnhandledError(null, new AirRuntimeError(rsp));
+  }
+  // General Air Service error
+  if (code === '3000') {
+    if (
+      rsp.faultcode !== undefined
+      && rsp.faultstring !== undefined
+      && /has no tickets/.test(rsp.faultstring) === true
+    ) {
+      return rsp;
+    }
+  }
+  switch (code) {
+    case '345':
+      throw new AirRuntimeError.NoAgreement({
+        pcc: utils.getErrorPcc(rsp.faultstring),
+      });
+    default:
+      throw new RequestRuntimeError.UnhandledError(null, new AirRuntimeError(rsp));
+  }
+}
+
+function airGetTickets(obj) {
+  // No tickets in PNR
+  if (
+    obj.faultcode !== undefined
+    && obj.faultstring !== undefined
+    && /has no tickets/.test(obj.faultstring) === true
+  ) {
+    return [];
+  }
+  // Parsing response
+  const tickets = airGetTicket.call(this, obj);
+  if (Array.isArray(tickets)) {
```

Comment:
Maybe it's better to declare the return value by using conditional operator rather than `if` statement. At least it will take less space.